handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
```

Comment:
Where does AuthenticationEnabled get set?  I see it declared, but I can't find where it is being set.  I'm assuming this is something that when we instantiate we should look at the server config passed in via the server?